### PR TITLE
Release v1.7.0

### PR DESCRIPTION
**Pull request by carlosalberto**
September release, following our monthly cadence.

@reyang Are we ready to go, metrics-wise?

@bogdandrutu @dyladan Should we include #1923 in this release?

cc @tigrannajaryan

### CODE REVIEW

**Comment by tigrannajaryan** [comment]
Looking at the changelog I don't see any changes to the schema so the schema file for 1.7.0 needs just an empty section for 1.7.0 version. @carlosalberto this should do it (put it in schemas/1.7.0):

```
file_format: 1.0.0
schema_url: https://opentelemetry.io/schemas/1.7.0
versions:
  1.7.0
  1.6.1:
  1.5.0:
  1.4.0:
```

**Comment by anuraaga** [comment]
Need action on #1658 before releasing I think @Oberon00 @bogdandrutu

**Comment by Oberon00** [comment]
Yes, please can we resolve #1658 now? It has been hanging around for quite a while and would be cumbersome to implement it if we don't release it in the same release that added the InstrumentationLibrary to the sampler.

**Comment by tigrannajaryan** [comment]
> Yes, please can we resolve #1658 now? It has been hanging around for quite a while and would be cumbersome to implement it if we don't release it in the same release that added the InstrumentationLibrary to the sampler.

I believe we are using the catch-the-train release model so #1658 is better merged soon or it is not going to make it :-)

**Review by tigrannajaryan — APPROVED** [review]

**Review by reyang — APPROVED** [review]
LGTM, thanks @carlosalberto!

**Comment by carlosalberto** [comment]
Hey, was wondering if we should postpone this release to the first days of October, so related issues are properly solved? Or would that hurt anybody?

**Comment by tigrannajaryan** [comment]
https://github.com/open-telemetry/opentelemetry-specification/pull/1658 is closed. Are we waiting for something else?

File: CHANGELOG.md [hunk 1]
```diff
@@ -11,6 +11,28 @@ release.
...
+
+### Semantic Conventions
+
+### Compatibility
+
+### OpenTelemetry Protocol
+
+### SDK Configuration
+
+## v1.7.0 (2021-09-15)
```
Comment (arminru) [inline CHANGELOG.md:28]: @carlosalberto Please make sure to check the changelog again before merging (so nothing is missing or slips into the wrong section) and also update the date here accordingly.

**Comment by carlosalberto** [comment]
Let's try to wait for #1970 by EOD, and then we finally do our release ;)

**Comment by tigrannajaryan** [comment]
> Let's try to wait for #1970 by EOD, and then we finally do our release ;)

@carlosalberto I will be travelling and may not be able to merge later today, please do it if you can.

**Pull request merged**